nodejs/package.json

```diff
@@ -18,15 +18,15 @@
         "test": "mocha test/*.js",
         "buildProto": "pbjs -t static-module -w commonjs -o gtfs-realtime.js ../gtfs-realtime.proto"
     },
-    "engines": [
-        "node >= 0.10.0"
-    ],
+    "engines": {
+        "node": "18.12.1"
+    },
     "dependencies": {
-        "protobufjs": "6.8.6"
+        "protobufjs": "6.11.3"
```

Comment:
As `npm upgrade` (and `yarn upgrade`) [respects SemVer ranges](https://docs.npmjs.com/cli/v9/commands/npm-update#description), there is no significant reason *not to use* ranges here (e.g. `^6.11.3`); `package-lock.json` will contain the exact version to be installed anyways.

This allows for the same tight control over dependency versions, while
- making minor & patch upgrades (which are expected to be non-breaking) much easier;
- reducing the noise of changes in `package.json`, which allows people to use the Git file history tools much better.